packages/admin_view/src/context/AuthContextProvider.tsx

```diff
@@ -0,0 +1,42 @@
+import { ReactNode, useCallback, useEffect, useState } from 'react';
+import { initializeApp } from 'firebase/app';
+import { getAuth, signInWithEmailAndPassword, signOut, onAuthStateChanged, User } from 'firebase/auth';
+import { AuthContext } from './auth-context';
+
+const AuthContextProvider = ({ children }: { children: ReactNode }) => {
+  const [token, setToken] = useState<string | null>(null);
```

Comment:
We won't actually want to store the JWT to use for auth. By default the JWT expires pretty quickly (1 hour I think by default). Instead, the `ApolloProviderWrapper` should call `user.getIdToken()` since that method checks for if the JWT is expired and refetches if needed

https://firebase.google.com/docs/reference/js/v8/firebase.User#getidtoken

Reply:
Ok. Removed the token from the context